utils/rtty/files/rtty.init

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh /etc/rc.common
+
+USE_PROCD=1
+START=99
+
+BIN=/usr/sbin/rtty
+
+start_service() {
+    host=$(uci get rtty.@server[0].host)
+    port=$(uci get rtty.@server[0].port)
+    interface=$(uci -q get rtty.@device[0].interface)
+    ifname=$(uci -p /var/state get network.$interface.ifname)
+    id=$(uci -q get rtty.@device[0].id)
+    description=$(uci -q get rtty.@device[0].description)
+    ssl=$(uci -q get rtty.@server[0].ssl)
+
+    procd_open_instance
+	procd_set_param command $BIN -h $host -p $port -a
+
+    [ -n "$interface" ] && {
+        ifname=$(uci -p /var/state get network.$interface.ifname)
+        procd_append_param command -i $ifname
+    }
```

Comment:
Please add a check `[ -n "$ifname" ]` and I guess the procd instance should not be opened before validation of this parameter.

Reply:
Fixed.